chart/values.schema.json

```
@@ -1979,6 +1979,45 @@
             "x-docsSection": "Jobs",
             "additionalProperties": false,
             "properties": {
+                "command": {
+                    "description": "Command to use when running create user job (templated).",
+                    "type": [
+                        "array",
+                        "null"
+                    ],
+                    "items": {
+                        "type": "string"
+                    },
+                    "default": null
+                },
+                "args": {
+                    "description": "Args to use when running create user job (templated).",
+                    "type": [
+                        "array",
+                        "null"
+                    ],
+                    "items": {
+                        "type": "string"
+                    },
+                    "default": [
+                        "bash",
+                        "-c",
+                        "exec \\\nairflow {{ semverCompare \">=2.0.0\" .Values.airflowVersion | ternary \"users create\" \"create_user\" }} \"$@\"",
+                        "--",
+                        "-r",
+                        "{{ .Values.webserver.defaultUser.role }}",
+                        "-u",
+                        "{{ .Values.webserver.defaultUser.username }}",
+                        "-e",
+                        "{{ .Values.webserver.defaultUser.email }}",
+                        "-f",
+                        "{{ .Values.webserver.defaultUser.firstName }}",
+                        "-l",
+                        "{{ .Values.webserver.defaultUser.lastName }}",
+                        "-p",
+                        "{{ .Values.webserver.defaultUser.password }}"
+                    ]
```

```suggestion
                    "default": "See values.yaml",
```

I almost did this, as this is a lot on the params page. Thoughts?

---

no objection from me.  but may be better to just remove the `default` key entirely (though perhaps that's not allowed with our docs build process / pre-commit?)

i looked at the params page, and it is pretty tough to navigate.  probably there is a better way to present this information.  and if that is done, maybe then it's more worthwhile to add.

Along these lines, from code dupe perspective, perhaps defaults could be scraped from values.yaml for the purpose of docs.

---

Yeah, our docs is "easy", but artifacthub uses the schema alone, e.g:

https://artifacthub.io/packages/helm/apache-airflow/airflow?modal=values-schema&path=workers.affinity

With fresh eyes, we probably should be removing "See values.yaml" instead of adding more.

And good callout on the params being rough... It does indeed need some tlc.